src/components/select-field/SelectField.scss

```diff
@@ -1,5 +1,6 @@
 @import '../../styles/variables';
 
+.bdl-SelectField,
 .select-field {
```

Comment:
Where is `.select-field` used? Should we delete it?

Reply:
We're not using `.select-field` in BUIE, but I found overrides for this class name in a couple of other projects.